build/sign.proj

```diff
@@ -13,8 +13,8 @@
     </CommonMSBuildProperties>
     <OutDir>$(RepositoryRootDirectory)</OutDir>
     <SignTargetsDependOn>BatchSign</SignTargetsDependOn>
-    <SignTargetsDependOn Condition="'$(SignVsix)' == 'true'">GetOutputVsix</SignTargetsDependOn>
-    <SignTargetsDependOn Condition="'$(SignNupkgs)' == 'true'">GetOutputNupkgs</SignTargetsDependOn>
+    <SignTargetsDependOn Condition="'$(SignPackages)' == 'true'">GetOutputNupkgs</SignTargetsDependOn>
+    <SignTargetsDependOn Condition="'$(SignPackages)' == 'true' AND '$(BuildRTM)' != 'true'">GetOutputNupkgs;GetOutputVsix</SignTargetsDependOn>
```

Comment:
Why this change?

    AND '$(BuildRTM)' != 'true'

Reply:
BuildRTM is a multiplier that runs two build in parallel for values true and false. When true, it produces nupkgs without the build number. In this build, the vsix is not produced, and hence the condition.